.github/workflows/pytest-gpu.yml

```diff
@@ -44,29 +45,34 @@ jobs:
 
         - name: pytest-gpu-acc
           test_file: "tests/test_gpu_openacc.py"
-          arch: "pgcc"
+          arch: "nvc"
           platform: "nvidiaX"
           language: "openacc"
           tags: ["self-hosted", "gpu", "openacc"]
 
     steps:
     - name: Checkout devito
-      uses: actions/checkout@v1
+      uses: actions/checkout@v2
 
     - name: Install dependencies
       run: |
-        pip3 install --upgrade pip
-        pip3 install -e .[extras]
+        # NOTE: Using 'source' during every step is ugly and should be changed
```

Comment:
This may help
https://github.com/devitocodes/devito/blob/6d2431c470cef3eb0f1dda4e961d8a8cc59333c9/.github/workflows/examples.yml#L42